simde/x86/svml.h

```diff
@@ -5588,6 +5565,122 @@ simde_mm512_mask_erfinv_pd(simde__m512d src, simde__mmask8 k, simde__m512d a) {
   #define _mm512_mask_erfinv_pd(src, k, a) simde_mm512_mask_erfinv_pd(src, k, a)
 #endif
 
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m128
+simde_mm_erfcinv_ps (simde__m128 a) {
+  #if defined(SIMDE_X86_SVML_NATIVE) && defined(SIMDE_X86_SSE_NATIVE)
+    return _mm_erfcinv_ps(a);
+  #elif SIMDE_NATURAL_VECTOR_SIZE_GE(128)
+    simde__m128 matched, retval = simde_mm_setzero_ps();
+
+    { /* if (a < 2.0f && a > 0.0625f) */
+      matched = simde_mm_and_ps(simde_mm_cmplt_ps(a, simde_mm_set1_ps(SIMDE_FLOAT32_C(2.0))), simde_mm_cmpgt_ps(a, simde_mm_set1_ps(SIMDE_FLOAT32_C(0.0625))));
+      retval = simde_mm_erfinv_ps(simde_mm_sub_ps(simde_mm_set1_ps(SIMDE_FLOAT32_C(1.0)), a));
+
+    }
+
+    { /* else if (a < 0.0625f && a >= 0.0f) */
+      simde__m128 mask = simde_mm_and_ps(simde_mm_cmplt_ps(a, simde_mm_set1_ps(SIMDE_FLOAT32_C(0.0625))), simde_mm_cmpge_ps(a, simde_mm_set1_ps(SIMDE_FLOAT32_C(0.0))));
+      mask = simde_mm_andnot_ps(matched, mask);
+      if (!simde_mm_test_all_zeros(simde_mm_castps_si128(mask), simde_x_mm_setone_si128())) {
+        matched = simde_mm_or_ps(matched, mask);
+
+        /* t =  1/(sqrt(-log(a))) */
+        simde__m128 t = simde_x_mm_negate_ps(simde_mm_log_ps(a));
+        t = simde_mm_sqrt_ps(t);
+        t = simde_mm_div_ps(simde_mm_set1_ps(SIMDE_FLOAT32_C(1.0)), t);
+
+        const simde__m128 p[] = {
+          simde_mm_set1_ps(SIMDE_FLOAT32_C( 0.1550470003116)),
+          simde_mm_set1_ps(SIMDE_FLOAT32_C( 1.382719649631)),
+          simde_mm_set1_ps(SIMDE_FLOAT32_C( 0.690969348887)),
+          simde_mm_set1_ps(SIMDE_FLOAT32_C(-1.128081391617)),
+          simde_mm_set1_ps(SIMDE_FLOAT32_C( 0.680544246825)),
+          simde_mm_set1_ps(SIMDE_FLOAT32_C(-0.164441567910))
+        };
+
+        const simde__m128 q[] = {
+          simde_mm_set1_ps(SIMDE_FLOAT32_C( 0.155024849822)),
+          simde_mm_set1_ps(SIMDE_FLOAT32_C( 1.385228141995)),
+          simde_mm_set1_ps(SIMDE_FLOAT32_C( 1.000000000000))
+        };
+
+        /* float numerator = p[0] / t + p[1] + t * (p[2] + t * (p[3] + t * (p[4] + t * p[5])))) */
+        simde__m128 numerator = simde_mm_fmadd_ps(p[5], t, p[4]);
+        numerator = simde_mm_fmadd_ps(numerator, t, p[3]);
+        numerator = simde_mm_fmadd_ps(numerator, t, p[2]);
+        numerator = simde_mm_fmadd_ps(numerator, t, p[1]);
+        numerator = simde_mm_add_ps(numerator, simde_mm_div_ps(p[0], t));
+
+        /* float denominator = (q[0] + t * (q[1] + t * (q[2]))) */
+        simde__m128 denominator = simde_mm_fmadd_ps(q[2], t, q[1]);
+        denominator = simde_mm_fmadd_ps(denominator, t, q[0]);
+
+         simde__m128 res = simde_mm_div_ps(numerator, denominator);
```

Comment:
There is an extra space at the beginning of the line; notice how it doesn't *quite* line up with the content above or below.